<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCircle</class>
 <widget class="QWidget" name="FormCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>圆</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormCircle{	
	background-color: rgb(174, 179, 188);
}

.QFrame{background-color: rgb(249, 249, 249);
border: 1px solid #555;
border-radius: 4px;
}

QSpinBox{
	min-height:35
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_left">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_param">
        <property name="title">
         <string>参数</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="1">
          <widget class="QSpinBox" name="spinBox_V_line_startX">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>高度</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinBox_height">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>竖划线X起点</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinBox_H_line_startY">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>横划线Y起点</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>圆心坐标X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>圆心坐标Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_CenterY">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_CenterX">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>圆半径R</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_radius">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>宽度</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="spinBox_width">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_btn">
        <property name="title">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_updateShow">
           <property name="text">
            <string>更新圆</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_next_HLine">
           <property name="text">
            <string>下一条横线</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_next_VLine">
           <property name="text">
            <string>下一条竖线</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_all_line">
           <property name="text">
            <string>所有线</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>278</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_right">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="horizontalSlider_scale">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>缩放</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_scale">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_center">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>定位到圆心</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_xy">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
